### PR TITLE
[Introspection] Render a comment for dbgenerated() default values into the schema

### PR DESCRIPTION
**Pull request by do4gr**
This will show the raw default value returned by the database. 

Fixes https://github.com/prisma/prisma/issues/3290

### CODE REVIEW

File: introspection-engine/introspection-engine-tests/tests/commenting_out/mod.rs [hunk 1]
```diff
@@ -227,3 +227,36 @@ async fn remapping_field_names_to_empty(api: &TestApi) -> crate::TestResult {
...
+        .await?;
+
+    let dm = indoc! {r##"
+        model Blog {
+          id            Int    @id @default(autoincrement())
+          number        Int?   @default(1)
+          /// This field's default value can currently not be parsed: `sqrt((4)::double precision)`.
+          bigger_number Int?   @default(dbgenerated())
+          // This type is currently not supported.
+          // This field's default value can currently not be parsed: `point((0)::double precision, (0)::double precision)`.
```
Comment (tomhoule) [inline introspection-engine/introspection-engine-tests/tests/commenting_out/mod.rs:254]: interesting that postgres "canonicalizes" the expression — I don't think mysql and sqlite do

File: introspection-engine/connectors/sql-introspection-connector/src/introspection_helpers.rs [hunk 2]
```diff
@@ -149,21 +149,35 @@ pub(crate) fn calculate_scalar_field(
...
+    let (is_commented_out, documentation) = match (&field_type, dbgenerated_string) {
+        (FieldType::Unsupported(_), None) => (true, Some("This type is currently not supported.".to_string())),
+        (FieldType::Unsupported(_), Some(default)) => (
+            true,
+            Some(
+                format!(
+                    "This type is currently not supported.\nThis field's default value can currently not be parsed: `{}`.",
+                    default
+                )
+                .to_string(),
```
Comment (tomhoule) [inline introspection-engine/connectors/sql-introspection-connector/src/introspection_helpers.rs:170]: nitpick: the `.to_string(),` is unnecessary, can be removed

**Review by tomhoule — APPROVED** [review]
:100:

File: introspection-engine/connectors/sql-introspection-connector/src/introspection_helpers.rs (hunk 2 above)
Comment (do4gr) [reply introspection-engine/connectors/sql-introspection-connector/src/introspection_helpers.rs]: Done

**Pull request merged**